manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -229,10 +236,12 @@ metadata:
     uid: "{{.UID}}"
 spec:
   ports:
-  - name: https-hbone
-    port: 15008
+  {{- range $key, $val := .Ports }}
```

Comment:
This is suspicious. Why do we need it?

Ingress exposes `status-port` on Service so Cloud LBs can health check it. Not sure why we need that for waypoint?

Reply:
Good point, I can revert this so we have 15008 only.

Reply:
Moving @costinm's reply here:
> We probably should - don't we want Cloud LBs to connect to waypoints on
their way to the app ? I don't think it's ideal to
have Cloud LB -> Gateway -> Waypoint -> Workload.

I think exposing all would fit more use-cases, but don't have a strong opinion on it at the moment. What do you think?

Reply:
We donot even have a service for waypoint previously, suggest supplement as needed